datahub-web-react/src/app/shared/ManageAccount.tsx

```diff
@@ -44,14 +45,16 @@ const defaultProps = {
 export const ManageAccount = ({ urn: _urn, pictureLink: _pictureLink, name }: Props) => {
     const entityRegistry = useEntityRegistry();
     const themeConfig = useTheme();
+    const { config } = useAppConfig();
     const handleLogout = () => {
         analytics.event({ type: EventType.LogOutEvent });
         isLoggedInVar(false);
         Cookies.remove(GlobalCfg.CLIENT_AUTH_COOKIE);
     };
-
+    const version = config?.appVersion;
     const menu = (
         <Menu>
+            {version && <MenuItem key="version">{version}</MenuItem>}
```

Comment:
can we make this link to release notes 🥺

Reply:
@dexter-mh-lee We can wrap this in a <Link to="url-to-release-notes"> component to do this^. Me or Gabe can also do it in a followup if its easier.